lib/client.js

```diff
@@ -914,6 +914,98 @@ BucketsMdapiClient.prototype.versionInternal = function _version(opts, cb) {
 };
 
 
+/**
+ * Gets a batch of garbage.
+ *
+ * @param {Object} opts              - request parameters
+ * @param {String} opts.request_id   - request uuid
+ * @param {Function} cb              - callback
+ */
+BucketsMdapiClient.prototype.getGarbageBatch =
```

Comment:
Naming nit: Why not match the rpc method name and call this `getGcBatch`? Ditto for the method below.

Reply:
Sure. I could do that if everyone else likes that better.